### PR TITLE
Should let enough time to tiller to come up

### PR DESCRIPTION
**Pull request by Zempashi**
and reduce CI time to failure (and avoid nasty garbage collection)

### CODE REVIEW

**Review by ballot-scality — DISMISSED** [review]

**Comment by Zempashi** [comment]
@bert-e reset

**Review by ballot-scality — DISMISSED** [review]

**Review by alexandre-allard — DISMISSED** [review]

File: ChangeLog.rst [hunk 1]
```diff
@@ -37,6 +37,8 @@ capital letters
 
 :ghpull:`472` - update Python `requests` library version
 
+:ghpull:`523` - reduce tiller wait timeout to reduce CI time to failure
```
Comment (alexandre-allard) [inline ChangeLog.rst]: Shouldn't Tiller be capitalized ?

Comment (Zempashi) [reply ChangeLog.rst]: Hmmm dunno maybe

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
@bert-e reset

**Comment by Zempashi** [comment]
conflict fixed

**Review by alexandre-allard — APPROVED** [review]

**Comment by Zempashi** [comment]
@bert-e approve

**Pull request merged**